components/BankPayouts/index.module.scss

```diff
@@ -0,0 +1,17 @@
+.input {
+    width: 412px;
+    height: 48px;
+
+    background: #FFFFFF;
+    /* Wireframe/Light Grey 2 */
+
+    border: 1px solid #DFDFDF;
```

Comment:
Please use the theme() function from tailwind to add colors.  Please check this out: https://tailwindcss.com/docs/using-with-preprocessors

Reply:
my bad, totally didn't remember.
sorted now!